ggshield/core/scan/commit_utils.py

```diff
@@ -193,48 +198,107 @@ def parse_patch(
     """
     Parse a patch generated with `git show` or `git diff` using PATCH_COMMON_ARGS.
 
-    If the patch represents a merge commit, then `patch` actually contains multiple
-    commits, one per parent, because we call `git show` with the `-m` option to force it
-    to generate one single-parent commit per parent. This makes later code simpler and
-    ensures we see *all* the changes.
+    patch looks like this:
```

Comment:
Nit: "A patch" ? 
Could also be good to add more details about what this function return, maybe something along the line of "this functions extracts the paths of files modified by the commit" ?